servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpClientBuildFinalizer.java

```diff
@@ -15,17 +15,41 @@
  */
 package io.servicetalk.http.api;
 
+import io.servicetalk.transport.api.ExecutionContext;
+
+@FunctionalInterface
 interface HttpClientBuildFinalizer {
+
+    /**
+     * Builds a new {@link StreamingHttpClient}, using a default {@link ExecutionContext}.
```

Comment:
Does `using a default {@link ExecutionContext}` part make sense? `HttpClientBuildFinalizer` doesn't know if users customized `ExecutionContext` or not.